package.json

```diff
@@ -33,8 +33,8 @@
     "prettier": "1.12.1",
     "request": "2.85.0",
     "semver": "5.5.0",
-    "sonarqube-scanner": "2.1.2",
-    "tfx-cli": "^0.7.6",
+    "sonarqube-scanner": "^2.5.0",
+    "tfx-cli": "^0.7.8",
```

Comment:
In sonarcloud-core repo we like to have fixed dependencies (without the `^` before the version) to be sure to never have bad surprises. I don't know if it make sens for this repo but I just wanted to raise the point :)

Reply:
Ok, done !